doc/tutorial/clean_mix.py

```diff
@@ -1,7 +1,7 @@
 from doit.task import clean_targets
 
 def simple():
-    print "ok"
```

Comment:
Crafting untested commits from regexes late at night will not work as intended :-(
Wil fix them manually.